website/src/pages/Blog/index.tsx

```diff
@@ -161,16 +161,24 @@ const Blog: React.FC = () => {
     );
   };
 
+  const handleTabClickOnMobile = (newValue: string) => {
+    setValue(newValue);
+    setPage(1);
+  }
```

Comment:
why this function? Its same as `handleTagSelect`

Reply:
Removed this function